security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,36 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-3-1"]
+== OpenShift Compliance Operator 1.3.1
+
+The following advisory is available for the OpenShift Compliance Operator 1.3.1:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5669[RHBA-2023:5669 - OpenShift Compliance Operator bug fix and enhancement update]
+
+This update addresses a CVE in an underlying dependency.
+//
+// This will be un-commented for OCP 4.11-4.14 releases. Commented out in the main branch.
+//
+// [IMPORTANT]
+// ====
+// It is recommended to update the Compliance Operator to version 1.3.1 or later before updating your {product-title} cluster to version 4.14 or later.
+// ====
+
+[id="compliance-operator-1-3-1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* You can install use the Compliance Operator in an {product-title} cluster in FIPS mode.
++
+{product-title} is designed for FIPS. When running {op-system-base-full} or {op-system-first} booted in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the x86_64, ppc64le, and s390x architectures.
```

Comment:
```suggestion
{product-title} is designed for FIPS. When running {op-system-base-full} or {op-system-first} running in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 validation on only the x86_64, ppc64le, and s390x architectures.
```

Reply:
I suggested lowercase "v" and  "running" vs. "booted". I don't have an argument against "booted", I just thought "running" was more accurate - your call sir.